CHANGES.txt

```diff
@@ -5,6 +5,15 @@ Changes for Crate
 Unreleased
 ==========
 
+- Updated crate-admin to ``1.1.0`` which includes the following changes:
+     - Breaking: Updated the admin-ui build to be compatible with future
```

Comment:
I think we can remove this note because it's internal and not relevant for users